src/tooltips/TooltipContainer.js

```diff
@@ -61,6 +62,7 @@ export default class TooltipContainer extends React.Component {
   renderTooltip() {
     const {hoverData, layerProps, theme} = this.props
     const {Tooltip = DefaultTooltip} = layerProps
+    const PositionedTooltip = withMeasuredSize(Tooltip, this.handleMeasure)
```

Comment:
This creates a new component in every render, which means that the tooltip needs to unmount and re-mount every render. Can we write the measuring as a component instead?

Reply:
✍️